bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/OpenWebNetBindingConstants.java

```diff
@@ -106,6 +116,17 @@
     // automation
     public static final String CHANNEL_SHUTTER = "shutter";
 
+    // thermo
+    public static final String CHANNEL_TEMPERATURE = "temperature";
+    public static final String CHANNEL_FUNCTION = "function";
+    // TODO REMOVE public static final String CHANNEL_HEATING_COOLING_MODE = "thermostatMode";
```

Comment:
TODO?

Reply:
sorry, it was an internal comment that I forgot to remove.